uint64 timestamp			# time since system start (microseconds)
uint64 ping_time			# Timestamp of the ping packet
uint32 ping_sequence		# Sequence number of the ping packet
uint32 dropped_packets		# Number of dropped ping packets
float32 rtt_ms				# Round trip time (in ms)
uint8 system_id				# System ID of the remote system
uint8 component_id			# Component ID of the remote system
